fabric-gametest-api-v1/src/main/java/net/fabricmc/fabric/impl/gametest/TestAnnotationLocator.java

```diff
@@ -82,7 +83,11 @@ private List<TestMethod> findMagicMethods(EntrypointContainer<Object> entrypoint
 	private void findMagicMethods(EntrypointContainer<Object> entrypoint, Class<?> testClass, List<TestMethod> methods) {
 		for (Method method : testClass.getDeclaredMethods()) {
 			if (method.isAnnotationPresent(GameTest.class)) {
-				validateMethod(method);
+				if (!CustomTestMethodInvoker.class.isAssignableFrom(testClass)) {
+					// Only validate the test method when using the default reflection invoker
+					validateMethod(method);
```

Comment:
Note: add the class name in validateMethod

Reply:
Another note : the warning in `findMagicMethods` mentions a `FabricGameTest` annotation, but the annotation's actual name is `GameTest`